substratevm/src/com.oracle.svm.reflect/src/com/oracle/svm/reflect/serialize/hosted/SerializationRegistry.java

```diff
@@ -23,7 +23,7 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-package com.oracle.svm.core.jdk.serialize;
+package com.oracle.svm.reflect.serialize.hosted;
```

Comment:
Do not move `SerializationRegistry` to a `*.hosted` package name. See https://github.com/oracle/graal/pull/3050/files#r554993045

Reply:
moved to parent package